var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -100,10 +100,11 @@ def cmake_args(self):
         spec = self.spec
 
         args = [
-            '-DBoost_ROOT_DIR=%s'     % spec['boost'].prefix,
-            '-DElfUtils_ROOT_DIR=%s'  % spec['elf'].prefix,
-            '-DLibIberty_ROOT_DIR=%s' % spec['libiberty'].prefix,
-            '-DTBB_ROOT_DIR=%s'       % spec['tbb'].prefix,
+            '-DBoost_ROOT_DIR=%s'       % spec['boost'].prefix,
+            '-DElfUtils_ROOT_DIR=%s'    % spec['elf'].prefix,
+            '-DLibIberty_ROOT_DIR=%s'   % spec['libiberty'].prefix,
+            '-DLibIberty_LIBRARIES=%s'  % spec['libiberty'].libs,
```

Comment:
This will probably fail if `libs` has more than one entry, since CMake will require semicolon-separated items. The problem is fixed transparently (and making the install script cleaner) by using
```suggestion
            self.define('LibIberty_LIBRARIES', spec['libiberty'].libs),
```
I recommend changing the other lines while you're at it, rather than manually matching the columns (which is discouraged in Python because it makes version control more difficult).